src/components/ui/CharInCircle.tsx

```diff
@@ -14,18 +15,19 @@ import classnames, {
   width,
 } from 'classnames/tailwind'
 
-const questionStyles = (size?: Sizes) =>
+const questionStyles = (size?: Sizes, disabled?: boolean) =>
   classnames(
     display('flex'),
     justifyContent('justify-center'),
     alignItems('items-center'),
     fontSize({ 'text-xs': Sizes.Small === size }),
-    fontFamily('font-primary')
+    fontFamily('font-primary'),
+    cursor({ 'cursor-pointer': !disabled })
   )
 
-const borderWrapper = (size?: Sizes, disabled?: boolean) =>
+const borderWrapper = (size?: Sizes) =>
   classnames(
-    boxSizing('box-border'),
+    boxSizing('box-content'),
```

Comment:
With `box-content` you will have a circle with the wrong sizes (`small`, for instance, will have 18x18 px. In the design docs the sizes are 16x16). Why you need `box-content` here?

Reply:
it's a mistake. ty, fixed it